react/week2/todolistsapp/src/components/Todotitle.jsx

```diff
@@ -0,0 +1,7 @@
+export default function Title() {
+    return (
+        <header style={{textAlign: "center", color: "orange"}}>
+            <h1>TODO LISTS</h1>
+        </header>
```

Comment:
It's good to separate the concerns in React, and create re-usable components as much as you can, but if you have a component which has a static text, and it's going to be used only once, or even twice, if it doesn't have any special styling or logic, why not just put the `h1` itself? since this has just added more code without adding any functionality.